application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -154,6 +157,17 @@ type Alert implements Node {
   trip: Trip @deprecated(reason : "Alert can have multiple affected entities now instead of there being duplicate alerts\nfor different entities. This will return only one of the affected trips.\nUse entities instead.")
 }
 
+"""
+Timing of an arrival or a departure to or from a stop. May contain real-time information if
+available.
+"""
+type ArrivalDepartureTime {
```

Comment:
Can you remind me why you're not reusing the type from the leg?

Reply:
I think the original reasoning was that the `scheduledTime` was non-null in the `LegTime` which might have caused issues with flex stuff. Additionally, the name of the type is a bit misleading. However, now that this type is only used from the "fixed" version of a stop time, `scheduledTime` probably could be non-null. Then again, there is this naming issue. I'm not sure if the solution is the rename this to be `FixedArrivalDepartureTime` or reuse the `LegTime` or not touch the name of this type (we decided to call the normal scheduled trip/stoptime fixed, why not this then?).

Reply:
I've marked some fields as non-null now in https://github.com/opentripplanner/OpenTripPlanner/pull/5393/commits/7ed48c3d556ab37bb375f8c23eb34018c30d5b39. I'll bring this topic up in the dev meeting today.

Reply:
We decided to not use the LegTime in schema. However, I have now slightly renamed this and renamed the internal LegTime class to match the new name and use it for both this and LegTime.